service/test/actors/TelemetryManagementActorTest.java

```diff
@@ -0,0 +1,97 @@
+package actors;
+
+import static org.junit.Assert.assertTrue;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import akka.testkit.javadsl.TestKit;
+import controllers.TelemetryControllerTest;
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.telemetry.actor.TelemetryManagerActor;
+import util.Constant;
+
+/**
+ * This class will have test cases for telemetry actor.
+ *
+ * @author Manzarul
+ */
+public class TelemetryManagementActorTest {
```

Comment:
1. Scenarios for different configurations: EkStep Telemetry - ON/OFF, kafka dispatcher - ON/OFF.

Reply:
did you add tests for these scenarios?